dash/frontend/src/app/modules/private/pages/cluster/kube-hunter/kube-hunter.component.ts

```diff
@@ -71,6 +72,11 @@ export class KubeHunterComponent implements OnInit, OnDestroy {
           this.allReportsForCluster = res.list;
           if (res.list[0] != null){
             this.daysPassed = Math.floor((Date.now() - res.list[0].createdAt) / (1000 * 60 * 60 * 24));
+            if (this.daysPassed >= 90){
+              this.ourAdvice = 'It has been ' + this.daysPassed + ' days since you ran Kube Hunter. You should run it again soon.';
+            } else {
+              this.ourAdvice = 'It has been ' + this.daysPassed + ' days since you last ran Kube Hunter. The current report is still valid.';
```

Comment:
Lets remove the "The current report is still valid" part. We don't know how frequently their security team wants them to run it.

Reply:
Oh, I see that's how it used to work, but yeah lets remove "The current report is still valid." anyways while we're working on this, thanks!

Reply:
Bump